onnx_tf/handlers/backend/resize.py

```diff
@@ -58,3 +145,115 @@ def process_NHWC_format(x):
                        lambda: process_NHWC_format(x))
 
       return [output]
+
+  @classmethod
+  def version_11(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    x = tensor_dict[node.inputs[0]]
+    x_shape = tf_shape(x)
+    roi = tensor_dict[node.inputs[1]]
+    scales = tensor_dict[node.inputs[2]]
+    sizes = tensor_dict[node.inputs[3]] if len(
+        node.inputs) == 4 else tf.constant([], dtype=tf.int64)
+    coordinate_transformation_mode = node.attrs.get(
+        "coordinate_transformation_mode", "half_pixel")
+    extrapolation_value = node.attrs.get("extrapolation_value", 0.0)
+    mode = node.attrs.get("mode", "nearest")
+
+    param = tf.cond(tf.greater(tf.size(scales), 0), lambda: scales,
+                    lambda: tf.cast(sizes, tf.float32))
+    n_in_param_is_one = tf.equal(param[0], 1)
+    c_in_param_is_one = tf.logical_or(tf.equal(param[1], 1),
+                                      tf.equal(param[3], 1))
+    assert_n_c_in_param_are_ones = tf.Assert(
+        tf.logical_and(n_in_param_is_one, c_in_param_is_one), [param])
+
+    with tf.control_dependencies([assert_n_c_in_param_are_ones]):
```

Comment:
Not sure I fully understand, this block seems to be necessary only if scales is provided. The spec says "If 'size' is needed, the user can use an empty string as the name of 'scales' in this operator's input list." Is it possible to check the third input name and skip some unnecessary ops? Not sure what values would be provided when input name is an empty string. So potentially the check might lead to unexpected results. If the this block is required even if only sizes has meaningful value, please add some comments.